Packs/HelloWorld/Integrations/FeedHelloWorld/FeedHelloWorld.py

```diff
@@ -0,0 +1,437 @@
+
+Note that the Client class should NOT contain any Cortex XSOAR specific code,
+i.e. it shouldn't use anything in the ``demisto`` class (functions such as
+``demisto.args()`` or ``demisto.results()`` or even ``return_results`` and
+``return_error``.
+You will use the Command Functions to handle XSOAR inputs and outputs.
+
+When calling an API, you should use the ``_http.request()`` method and you
+can return the raw data to the calling function (usually a Command function).
+
+You should usually have one function for each API endpoint.
+
+Look at the code and the commands of this specific class to better understand
+the implementation details.
+
+
+Helper Functions
+----------------
+
+Helper functions are usually used as utility functions that are used by several
+command functions throughout your code. For example they map arguments to types
+or convert severity formats from feed integration-specific to XSOAR.
+Many helper functions are already defined in ``CommonServerPython.py`` and are
+often very handy.
+
+
+Command Functions
+-----------------
+
+Command functions perform the mapping between XSOAR inputs and outputs to the
+Client class functions inputs and outputs. As a best practice, they shouldn't
+contain calls to ``demisto.args()``, ``demisto.results()``, ``return_error``
+and ``demisto.command()`` as those should be handled through the ``main()``
+function.
+However, in command functions, use ``demisto`` or ``CommonServerPython.py``
+artifacts, such as ``demisto.debug()`` or the ``CommandResults`` class and the
+``Common.*`` classes.
+
+Every feed integration should have these three base commands:
+``<product-prefix>-get-indicators`` - where <product-prefix> is replaced by the name
+of the Product or Vendor source providing the feed. So for example, if you were
+developing a feed integration for Microsoft Intune this command might be called
+msintune-get-indicators. This command should fetch a limited number of indicators
+from the feed source and display them in the war room.
+``fetch-indicators`` - this command will initiate a request to the feed endpoint, format
+the data fetched from the endpoint to conform to Cortex XSOAR's expected input format
+and create new indicators. If the integration instance is configured to Fetch indicators,
+then this is the command that will be executed at the specified Feed Fetch Interval.
+``test-module`` - this is the command that is run when the Test button in the configuration
+ panel of a feed integration is clicked.
+
+
+
+Command functions, when invoked through an XSOAR command usually return data
+using the ``CommandResults`` class, that is then passed to ``return_results()``
+in the ``main()`` function.
+``return_results()`` is defined in ``CommonServerPython.py`` to return
+the data to XSOAR. ``return_results()`` actually wraps ``demisto.results()``.
+You should never use ``demisto.results()`` directly.
+
+Sometimes you will need to return values in a format that is not compatible
```

Comment:
I don't think this is relevant for a Feed Integration

Reply:
I would probably remove also some of the paragraphs below as they are not relevant for feed integrations. If possible, try to keep it simple and focused to the task at hand.

Reply:
Done